LibertyDSNP/frequency
# Config Treasury

# Goal
The goal of this PR is to setup the treasury pallet

Closes #348

# Discussion
- How to Test:
  - Start the chain with the spend period once every 2 blocks: `FRQCY_SPEND_PERIOD=2 make start`
  - Create a proposal
  - Commit it to council
  - Vote on Proposal (2x)
  - Close proposal
  - (Now the proposal is just waiting for the end of the spend period)
  - `make local-block` 2x
  - See the proposal gone, and the money transferred

# Checklist
- [x] Updated the js/api-augment code if a custom RPC added/changed
- [x] N/A Design doc(s) updated
- [x] Manual Testing. Automated e2e tests tbd
- [ ] Weights updated (triggering Benchmarks after all the config stories are in)
